control-plane-operator/controllers/hostedcontrolplane/configoperator/reconcile.go

```diff
@@ -114,9 +114,23 @@ func ReconcileRole(role *rbacv1.Role, ownerRef config.OwnerRef) error {
 			APIGroups: []string{corev1.SchemeGroupVersion.Group},
 			Resources: []string{
 				"secrets",
+			},
+			Verbs: []string{
+				"create",
```

Comment:
why did you add "create secrets" permission?
I don't see any new secrets creation in the new code.

Reply:
ah, good catch. I don't think we need `create`.